.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,37 @@
+name: ci/cd
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
why do you want all history for the CI job?

Reply:
I copied it from the evy cicd.yaml. maybe firebase-deploy needed that, in which case i'll just remove it here.